src/Api/ItemList.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace OpsWay\ZohoBooks\Api;
+
+
+class ItemList implements \ArrayAccess, \IteratorAggregate
+{
+    /**
+     * @var iterable
+     */
+    private $items = [];
+
+    /**
+     * @var iterable
+     */
+    private $pageContext = [];
+
+    function __construct(iterable $items,iterable $pageContext)
+    {
+        $this->items = $items;
+        $this->pageContext = $pageContext;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getIterator()
+    {
+        return new \ArrayIterator($this->items);
+    }
+
+    /**
+     * @return iterable
+     */
+    public function getPageContext(): iterable
```

Comment:
Same as comment above, now with PHP 5.6 support we can not use return type hinting. Also in future we just add it without BC (iterable included array type).